web/templates/layout/app.html.eex

```diff
@@ -16,6 +16,14 @@
   </head>
 
   <body class="w-100 sans-serif bg-white">
+
+    <header class="hl-bg-aqua w-100 ph3 pv3 pv4-ns ph4-m ph5-l tc">
+      <nav class="f4 fw3 tracked">
+        <i id="open-nav" class="fa fa-bars dim white dib fl" aria-hidden="true"></i>
+        <a href="/"><%= img_tag("/images/logo.svg", class: "w-70") %></a>
```

Comment:
Cool, I didnt do a fallback, I just changed the image to png instead. 

Are we doing a check for whether its IE8 or not? I can apply the trick of:
`<svg width="96" height="96">
  <image xlink:href="image.svg" src="image.png" width="96" height="96" />
</svg>`

but not sure how to do it in Phoenix...